### PR TITLE
Have templates be replacable with endpoints

### PR DESCRIPTION
**Pull request by bbenligiray**
Closes https://github.com/api3dao/airnode-protocol-v1/issues/35

### CODE REVIEW

**Review by acenolaza — APPROVED** [review]
LGTM 👍🏻

File: contracts/dapis/DapiServer.sol [hunk 1]
```diff
@@ -208,7 +247,50 @@ contract DapiServer is
...
+    }
+
+    /// @notice Creates an RRP request for the Beacon to be updated by the relayer
+    /// @param airnode Airnode address
+    /// @param endpointId Endpoint ID
+    /// @param parameters Parameters
+    /// @param relayer Relayer address
+    /// @param sponsor Sponsor address
+    /// @return requestId Request ID
+    function requestRelayedRrpBeaconUpdateWithEndpoint(
```
Comment (Ashar2shahid) [inline contracts/dapis/DapiServer.sol:268]: I feel like this function is just bloat

**Review by Ashar2shahid — APPROVED** [review]
Looks good to me, just a comment

File: contracts/dapis/DapiServer.sol (hunk 1 above)
Comment (bbenligiray) [reply contracts/dapis/DapiServer.sol:268]: Worst case is it's for the sake of completeness, but I can actually see it being a thing in the future

**Pull request merged**